Firebase/Database/Api/FIRDatabaseComponent.m

```diff
@@ -24,16 +24,16 @@
 #import <FirebaseCore/FIRAppInternal.h>
 #import <FirebaseCore/FIRComponent.h>
 #import <FirebaseCore/FIRComponentContainer.h>
-#import <FirebaseCore/FIRComponentRegistrant.h>
 #import <FirebaseCore/FIRDependency.h>
+#import <FirebaseCore/FIRLibrary.h>
 #import <FirebaseCore/FIROptions.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 /** A NSMutableDictionary of FirebaseApp name and FRepoInfo to FirebaseDatabase instance. */
 typedef NSMutableDictionary<NSString *, FIRDatabase *> FIRDatabaseDictionary;
 
-@interface FIRDatabaseComponent () <FIRComponentLifecycleMaintainer, FIRComponentRegistrant>
+@interface FIRDatabaseComponent () <FIRComponentLifecycleMaintainer, FIRLibrary>
```

Comment:
In a follow up PR we can probably integrate the `FIRComponentLifecycleMaintainer` as well.

Reply:
Sounds good.